src/components/BannerNotification/BannerNotification.stories.tsx

```diff
@@ -1,28 +1,34 @@
 import React from "react"
 import { useTranslation } from "next-i18next"
 import { Text } from "@chakra-ui/react"
-import { Meta, StoryFn } from "@storybook/react"
+import { Meta, StoryObj } from "@storybook/react"
 
 import BannerNotification from "."
 
-export default {
+const meta = {
+  title: "PostMergeBanner",
   component: BannerNotification,
-  args: {
-    shouldShow: true,
-  },
-  decorators: [(Story) => <Story />],
-} as Meta<typeof BannerNotification>
+} satisfies Meta<typeof BannerNotification>
+
+export default meta
 
 /**
  * Story taken from PostMergeBanner component
  * and content from `../../content/developers/tutorials/hello-world-smart-contract-fullstack/index.md`
  */
-export const PostMergeBanner: StoryFn<typeof BannerNotification> = (args) => {
-  const { t } = useTranslation("page-upgrades")
 
-  return (
-    <BannerNotification {...args}>
-      <Text>{t("page-upgrades-post-merge-banner-tutorial-ood")}</Text>
-    </BannerNotification>
-  )
+export const PostMergeBanner: StoryObj<typeof meta> = {
+  args: {
+    shouldShow: true,
+  },
+  render: (args) => {
+    // eslint-disable-next-line react-hooks/rules-of-hooks
+    const { t } = useTranslation("page-upgrades")
```

Comment:
Oh interesting, I wasn't sure we could use hooks in here, good to know 🙏

Reply:
Storybook does not seem to like this, where better would be to make a wrapper component to then apply to this render. 

But this is quicker 😅